driver/chain_syncer/calldata/syncer_test.go

```diff
@@ -79,6 +79,8 @@ func (s *CalldataSyncerTestSuite) TestCancelNewSyncer() {
 	defer s.cancel()
```

Comment:
is this necessary? since i also see `cancel` in line 81

Reply:
unnecessary, removed